ta4j-core/pom.xml

```diff
@@ -18,6 +18,12 @@
             <artifactId>slf4j-api</artifactId>
             <version>1.7.25</version>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.2.3</version>
+            <scope>test</scope>
+		</dependency>
```

Comment:
> library should not have a logger implementation as dependency. This should be up to the user

[first commit only](https://github.com/ta4j/ta4j/pull/198#issuecomment-369730379)



> Is it not possible to add this functionality to the existing BigDecimalNum instead of creating a new implementation with the same delegate?

[I'm concerned about the heap overhead](https://github.com/ta4j/ta4j/pull/198#issuecomment-370167986) but I have not made a quantitative assessment yet.